common/coat_of_arms/coat_of_arms/wc_religious_titles.txt

```diff
@@ -0,0 +1,35 @@
+### Religious titles ###
+
+### Holy Light
+d_church_of_the_holy_light = {
+	pattern = "pattern_solid.dds"
+	color1 = "white"
+	colored_emblem = {
+		texture = "ce_holy_light.dds"
+		color1 = "yellow"
+		color2 = "yellow"
+		instance = { position = { 0.5 0.4 } scale = { 0.9 0.9 }  }
```

Comment:
Remember we talked about better not to use `instance` and correct the image instead.

Reply:
fixed in a359075d0